src/main/kotlin/controller/BlackJackController.kt

```diff
@@ -0,0 +1,62 @@
+package controller
+
+import model.Dealer
+import model.Player
+import service.ResultCalculator
+import util.isYes
+import view.InputView
+import view.OutputView
+
+object BlackJackController {
+    fun playGame() {
+        val players = getPlayerNames().map { Player(it) }
+        val dealer = Dealer()
+        startGame(players, dealer)
+        runTurns(players, dealer)
+        showResults(players, dealer)
+    }
+
+    private fun getPlayerNames(): List<String> {
+        val players = InputView.requestPlayerNames()
+        require(players.size <= 6) { "Maximum player names must be 6" }
+        return players
+    }
```

Comment:
I think the player count limit is also part of the business logic.

What do you think about moving it into a separate class to manage it more clearly?
It could be a good opportunity to introduce an `Players` class as a first-class collection!